root/layout/components/NewFooter.js

```diff
@@ -0,0 +1,242 @@
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://wiki.musicbrainz.org/Main_Page">
+                  {l('Wiki')}
+                </a>
+              </li>
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://community.metabrainz.org/">
+                  {l('Community')}
+                </a>
+              </li>
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://blog.metabrainz.org/">
+                  {l('Blog')}
+                </a>
+              </li>
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://www.redbubble.com/people/metabrainz/shop">
+                  {l('Shop')}
+                </a>
+              </li>
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://metabrainz.org/">
+                  {l('MetaBrainz')}
+                </a>
+              </li>
+              <li className="d-block d-md-none">
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://github.com/metabrainz/listenbrainz-server">
+                  {l('Contribute Here')}
+                </a>
+              </li>
+              <li className="d-block d-md-none">
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://tickets.metabrainz.org/">
+                  {l('Bug Tracker')}
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div className="col-12 col-md-4 col-lg-3">
+            <h3 className="text-brand text-body">{l('Fellow Projects')}</h3>
+            <ul className="list-unstyled">
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://listenbrainz.org/">
+                  {l('ListenBrainz')}
+                </a>
+              </li>
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://critiquebrainz.org/">
+                  {l('CritiqueBrainz')}
+                </a>
+              </li>
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://picard.musicbrainz.org/">
+                  {l('Picard')}
+                </a>
+              </li>
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://bookbrainz.org/">
+                  {l('BookBrainz')}
+                </a>
+              </li>
+              <li>
+                <FontAwesomeIcon icon={faAngleRight} size="lg" />
+                <a href="https://acousticbrainz.org/">
+                  {l('AcousticBrainz')}
+                </a>
+              </li>
```

Comment:
Probably best not to link to this anymore? We did after all close it :) If you very much want to keep the number of projects the same as the other columns, add the Event Art Archive (https://eventartarchive.org/)